spec/index.html

```diff
@@ -761,12 +761,12 @@ <h4>rdf:predicate</h4>
         of S is P.</p>
       <p>The <a href="#ch_domain"><code>rdfs:domain</code></a> of
         <code>rdf:predicate</code> is <code><a href="#ch_statement">rdf:Statement</a></code>
-        and the <a href="#ch_range"><code>rdfs:range</code></a> is <a href="#ch_resource"><code>rdfs:Resource</code></a>.</p>
+        and the <a href="#ch_range"><code>rdfs:range</code></a> is <a href="#ch_resource"><code>rdfs:Property</code></a>.</p>
```

Comment:
```suggestion
        and the <a href="#ch_range"><code>rdfs:range</code></a> is <a href="#ch_resource"><code>rdfs:Resource</code></a>.</p>
```

My take away from the email discussion was that the range of rdf:predicate (`rdf:Resource`) was in fact correct, and that the misleading sentence "states that S is..." that seemed to imply a different range (`rdf:Property`) was to be fixed by removing "that P is an instance of `rdf:Property`".

See https://lists.w3.org/Archives/Public/public-rdf-comments/2021Dec/0005.html (last email of the thread).

With the propose (un)change above, this PR is limited to its "small HTML fixes", and is therefore purely editorial.